packages/pwa/app/utils/utils.js

```diff
@@ -154,3 +156,9 @@ export const capitalize = (text) => {
         .map((s) => s.charAt(0).toUpperCase() + s.substring(1))
         .join(' ')
 }
+
+/**
+ * A util to return current url configuration
+ * @returns {{locale: string, siteId: string}}
+ */
+export const getUrlConfig = () => pwaKitConfig.app.url
```

Comment:
If someone messes around with their config you might run into null pointer exceptions. You can easily fix this with using the `?` optional syntax.